ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2137,6 +2137,10 @@ router_bgp:
         - < learned >
   address_family_evpn:
     domain_identifier: < string >
+    neighbor_default_encapsulation:
+      mpls: < true | false >
```

Comment:
We have two options, `vxlan` or `mpls`. So we should have an option with these as the possible values instead.
There are also other options on `neighbor default` so we should change the data model a bit. Proposal:
```yaml
  address_family_evpn:
    neighbor_default:
      encapsulation: < vxlan | mpls >
      mpls_next_hop_self_source_interface: < source_interface >
```

Reply:
Aligned data model to your request.

Reply:
Fixed.